packages/core/src/components/graphs/wordcloud.ts

```diff
@@ -51,18 +51,27 @@ export class WordCloud extends Component {
 			return;
 		}
 
+		const wordCloudData = [];
+		displayData.map(function (d) {
+			let value = d[fontSizeMapsTo];
+
+			if (typeof d[fontSizeMapsTo] !== 'number') {
+				throw Error(
+					'Badly formatted WordCloud data. `value` should only be an integer or float'
+				);
+			}
+
+			wordCloudData.push({
+				[groupMapsTo]: d[groupMapsTo],
+				text: d[wordMapsTo],
+				size: value,
+				value,
+			});
+		});
+
 		const layout = cloud()
 			.size([width, height])
-			.words(
-				displayData.map(function (d) {
-					return {
-						[groupMapsTo]: d[groupMapsTo],
-						text: d[wordMapsTo],
-						size: d[fontSizeMapsTo],
-						value: d[fontSizeMapsTo],
-					};
-				})
-			)
+			.words(wordCloudData)
```

Comment:
```suggestion
			.words(
				displayData.map(function (d) {
				   // Add check here & throw error if true. Otherwise continue? Error should be displayed instead of the render, correct?
				   if (typeof d[fontSizeMapsTo] !== 'number') {
				      throw Error(
					    'Badly formatted WordCloud data. `value` should only be an integer or float'
				      );
			       }
			       
					return {
						[groupMapsTo]: d[groupMapsTo],
						text: d[wordMapsTo],
						size: d[fontSizeMapsTo],
						value: d[fontSizeMapsTo],
					};
				})
			)
```